.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: lint
+on: [push]
+jobs:
+  app:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: app
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Read .nvmrc
```

Comment:
By the way, per [Specifying a Node Version](https://render.com/docs/node-version):

> By default, Render uses the latest LTS version of Node.

which is 12.16.3 at the moment. There are different ways to configure that, none of which are `.nvmrc`, but we should probably try to keep them in sync?

Reply:
I'm cool just leaving it for now 🤷‍♂️